manage.py

```diff
@@ -3,8 +3,5 @@
 import sys
 
 if __name__ == "__main__":
```

Comment:
Reduce locations where environment variables are set. For now, limit to host environment, settings.py, local_settings.py, docker-compose and Dockerfiles. By starting to consolidate and extract from various locations, such as manage.py we can make it easier to parameterize